### PR TITLE
Feat add react admin

### PR DESCRIPTION
**Pull request by Arfey**

### CODE REVIEW

File: aiohttp_admin/templates/admin_on_rest.jinja2 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {% block head %}
+    <meta charset="UTF-8">
+    <title>{{ name }} Admin</title>
+    {% endblock %}
+    <!-- main application data -->
+    <script type="text/javascript">
+        window.appData = {
+          tokenUrl: "{{ url('admin.token') }}",
+          tokenName: "aiohttp_admin_token",
+          loginUrl: "{{ url('admin.token') }}",
```
Comment (jettify) [inline aiohttp_admin/templates/admin_on_rest.jinja2]: what the difference between loginUrl and tokenUrl ?

File: aiohttp_admin/admin.py [hunk 2]
```diff
@@ -80,3 +83,62 @@ def setup_admin_handlers(admin, admin_handler, static_folder, admin_conf_path):
...
+
+        for r in resources:
+            r.setup(self._admin, URL('/'))
+        self._resources = tuple(resources)
+
+    @property
+    def resources(self):
+        return self._resources
+
+    async def index_page(self, request):
```
Comment (jettify) [inline aiohttp_admin/admin.py]: Lets add at least one test case for this handle

**Comment by jettify** [comment]
This task is first one for https://github.com/aio-libs/aiohttp_admin/issues/359

File: aiohttp_admin/templates/admin_on_rest.jinja2 (hunk 1 above)
Comment (Arfey) [reply aiohttp_admin/templates/admin_on_rest.jinja2]: I could not choose which is better) thx i will remove some

File: aiohttp_admin/admin.py (hunk 2 above)
Comment (Arfey) [reply aiohttp_admin/admin.py]: \+

**Comment by jettify** [comment]
thanks!

**Pull request merged**